バイナリー・ファイルCanna37p1.orig/lib/RKC/.wconvert.c.swpとCanna37p1/lib/RKC/.wconvert.c.swpは違います
diff -ruN Canna37p1.orig/lib/RKC/conf.c Canna37p1/lib/RKC/conf.c
--- Canna37p1.orig/lib/RKC/conf.c	2003-12-28 02:15:24.000000000 +0900
+++ Canna37p1/lib/RKC/conf.c	2004-02-19 22:24:48.882713989 +0900
@@ -1496,7 +1496,7 @@
 };
 
 const NumberDefaultRec host_num_defaults[] = {
-  { CONF_SERVER_TIMEOUT, 1500u },
+  { CONF_SERVER_TIMEOUT, 0u },
 };
 
 const char *
